postZephir.pm

```diff
@@ -467,11 +464,11 @@ sub main {
       };
     
       my $sdr_source = lc($collection);
-      $source eq 'MiU' and $sdr_source = 'miu';
+      lc($source) eq 'miu' and $sdr_source = 'miu';
```

Comment:
In the records the source is generally set to `MIU`, not `MiU`. Might be an alma change. Either way I'm not certain why there is special handling for this, or why we're checking this at all here, and there is a ticket to remove it: DEV-457